pi5/board.go

```diff
@@ -82,15 +82,18 @@ func newBoard(
 
 		// store addresses + other stuff here
 		gpioNodePath: "",
-		mem_fd:       0,
-		phys_addr:    &physical_address,
-		virt_addr:    &virtual_address,
-		gpiomem_idx:  gpiomem_idx,
+		memFD:        0,
+		physAddr:     &physical_address,
```

Comment:
i dont understand why you need the `physical_address`, is there a reason why you can't just assign the `physAddr `field once its determined? same question for` virtual_address`

Reply:
I'm not sure if I need the physical & virtual address to be pointers or not yet. I though I would need a pointer to the physical address for mmaping, but It works without so I can changed this